Dockerfile

```diff
@@ -3,15 +3,15 @@
 
 # Should be the specific version of `node:slim`.
 # `sqitch` requires at least `buster`.
-FROM node:19.7.0-slim AS development
+FROM node:19.7.0 AS development
```

Comment:
We should not remove `slim` in this PR and try to resolve the issue separately.

Reply:
Alright